app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: int) -> int:
```

Comment:
The parameter `team` is annotated as `int`, but it seems to be used as an iterable in the function. Consider changing the annotation to a more appropriate type, such as `list` or `Iterable`.